hw/xbox/nv2a/nv2a.c

```diff
@@ -104,7 +101,10 @@ void *nv_dma_map(NV2AState *d, hwaddr dma_obj_address, hwaddr *len)
 
     /* TODO: Handle targets and classes properly */
     NV2A_DPRINTF("dma_map %x, %x, %" HWADDR_PRIx " %" HWADDR_PRIx "\n",
-                 dma.dma_class, dma.dma_target, dma.address, dma.limit);
+                 dma.dma_class,
+                 dma.dma_target,
+                 dma.address,
+                 dma.limit);
```

Comment:
Not sure if I like this change. I prefer to keep `NV2A_DPRINTF` short and use newlines to highlight specific semantics (`x, y, z,` of a vector on the same line for example)

Reply:
Still not sure why clang-format packs them sometimes but not others. I'm okay to have them on one line as well.